dashboard/settings.py

```diff
@@ -309,9 +310,6 @@
 # TODO: Select a default course for user from database, just here so things don't all break
 DEFAULT_COURSE_IDS = config("CANVAS_COURSE_IDS", default="0", cast=Csv())
```

Comment:
There are two entries in the .env.sample for "CANVAS_COURSE_IDS". Can you also remove one there?

Reply:
Sure! These "DEFAULT_COURSE_IDS" now are really just the default id's that are populated by the cron. This is no longer used otherwise after this change.